Common/Models/PumpManagerUI.swift

```diff
@@ -16,7 +16,7 @@ private let managersByIdentifier: [String: PumpManagerUI.Type] = staticPumpManag
 
 typealias PumpManagerHUDViewsRawValue = [String: Any]
 
-func PumpManagerHUDViewsFromRawValue(_ rawValue: PumpManagerHUDViewsRawValue, pluginManager: PluginManager) -> [BaseHUDView]? {
+func PumpManagerHUDViewsFromRawValue(_ rawValue: PumpManagerHUDViewsRawValue, pluginManager: PluginManager) -> [LevelHUDView]? {
```

Comment:
If this is just going to vend a single reservoir view, we should change the method names to reflect that.